File: inJava/app/src/main/java/com/barrygithub/rickandmortyapp/data/LocalDatasource/MyDatabase.java
package com.barrygithub.rickandmortyapp.data.LocalDatasource;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.barrygithub.rickandmortyapp.data.LocalDatasource.entities.Character;
import com.barrygithub.rickandmortyapp.data.LocalDatasource.entities.MetaData;


/**
 * Project RickAndMortyApp
 * Created by Barry Zea H. on 5/04/23.
 * Copyright (c) Barry Zea H. All rights reserved.
 **/
@Database(entities = {Character.class, MetaData.class}, version = 1)
public abstract class MyDatabase extends RoomDatabase {
    public abstract MyDAO myDAO();
}

File: inKotlin/app/src/main/java/com/barrygithub/rickandmortyapp/data/localDatasource/MyDatabase.kt
package com.barrygithub.rickandmortyapp.data.localDatasource

import androidx.room.Database
import androidx.room.RoomDatabase

/**
 * Project RickAndMortyApp
 * Created by Barry Zea H. on 14/04/23.
 * Copyright (c) Barry Zea H. All rights reserved.
 *
 **/
@Database(entities = [Character::class, MetaData::class], version = 1)
abstract class MyDatabase: RoomDatabase() {
    abstract fun myDAO():MyDAO
}